ngrinder-controller/src/main/java/org/ngrinder/compatibility/perftest/controller/PerfTestFormTypeApiController.java

```diff
@@ -35,8 +34,12 @@
 @Deprecated
 public class PerfTestFormTypeApiController extends PerfTestApiController {
```

Comment:
@RequireArgsConstructor

Reply:
`PerfTestFormTypeApiController` is child class from  `PerfTestApiController`.

@RequireArgsConstructor can't call the parent constructor.

ref: https://stackoverflow.com/a/29771875/8074178
related issue: https://github.com/rzwitserloot/lombok/issues/375